scipy/special/_basic.py

```diff
@@ -2176,6 +2177,18 @@ def comb(N, k, exact=False, repetition=False):
     repetition : bool, optional
         If `repetition` is True, then the number of combinations with
         repetition is computed.
+    legacy : bool, optional
+        If `legacy` is True and `exact` is True, then non-integral arguements
+        are cast to ints, otherwise non-integral arguements result in the
+        calculation being performed as if `exact` is False.
```

Comment:
I don't really like this suggested change here. It may be slicker but I find the original easier to understand. I think "independent" has too much baggage within statistics and mathematics, making its use here sound strange to my ears.

Reply:
Is it just about the word "independent"? That would be easily replaceable ("[...] the result for non-integral arguments is unaffected by the value of `exact`.")

I'm thinking about non-native speakers and I find the negation in "If X and Y <behavior_A>, otherwise <behavior_B>" much more difficult to process correctly than "If X and Y <behaviour_A>; if not X <behaviour_B>".

Reply:
Yeah, I think that's it. "unaffected by the value of exact" sounds fine to me.